packages/deployment/app/cloudformation.template.yml

```diff
@@ -700,6 +700,12 @@ Resources:
           {{@key}}: {{{this}}}
       {{/if}}
     {{/each}}
+      {{# if ../api_backend_url}}
+          BACKEND_API_URL: {{../api_backend_url}}
```

Comment:
This change will set `BACKEND_API_URL` as an environment variable in all lambdas deployed by Cumulus. I don't think this is an issue, but I'm interested to hear others' opinions.

Reply:
seems useful to me. but what do I know?